addons/web_editor/static/lib/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -143,7 +144,9 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false,
                     return;
                 }
             }
-            parentEl.oDeleteBackward(parentOffset, alreadyMoved);
+            if(!(this.tagName === 'A' && !this.classList.contains('btn') && !isVisible(this))) {
+                parentEl.oDeleteBackward(parentOffset, alreadyMoved);
+            }
```

Comment:
@adsa-odoo  instead of too many negation we can simply use:
`if (this.tagName !== 'A' || this.classList.contains('btn') || isVisible(this)) {
    parentEl.oDeleteBackward(parentOffset, alreadyMoved);
}`

Reply:
but i guess it will not meet the condition for which we might be preventing the case, correct?

Reply:
Okay! Got it.